cmd/scheduler/app/options/options.go

```diff
@@ -32,6 +32,7 @@ import (
 const (
 	defaultSchedulerName   = "volcano"
 	defaultSchedulerPeriod = time.Second
+	defaultResyncPeriod    = 5 * time.Minute
```

Comment:
Can you give more details about why we need this resync perion, as I didn't see that the resync is set in upstream kubernetes components: )

Reply:
It is too small and will cause scheduler performance issue.

Reply:
I think we not disable it (set to zero) by default and users can set a positive value as needed.